yazi-boot/src/args.rs

```diff
@@ -6,8 +6,8 @@ use clap::{command, Parser};
 #[command(name = "yazi")]
 pub struct Args {
 	/// Set the current working entry
-	#[arg(index = 1)]
-	pub entry: Option<PathBuf>,
+	#[arg(index = 1, value_delimiter = ' ', num_args = 1..)]
```

Comment:
Please limit the maximum number of parameters to 9 as Yazi can only create up to 9 tabs.

Reply:
done.